README.md

```diff
@@ -108,6 +109,29 @@ as generate migrations and an authorization module in your `lib/thesis_auth.ex`.
 ```
 $ mix ecto.migrate
 ```
+
+##### Page content caching
+
+To enable content caching, add this in your config/config.exs:
```

Comment:
Maybe flesh out the paragraph here, give a little more background regarding why this is desirable.